internal/execute/outputs.go

```diff
@@ -244,9 +245,9 @@ func generateGroupOptionOutput(sys System, optionsList []*tsoptions.CommandLineO
 	}
 
 	// make sure always a blank line in the end.
-	// !!! if lines[len(lines)-2] != sys.NewLine() {
-	// !!! 	lines = append(lines, sys.NewLine())
-	// !!! }
+	if lines[len(lines)-2] != sys.NewLine() {
```

Comment:
Can this ever go out of range?

Reply:
Accessing an array element that's `out of range` in TS/JS returns `undefined`, so to align with `strada`, tsgo needs to insert a newline when the length is less than 2.